Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -4493,6 +4496,7 @@ public void Actions_WhenFindingControlSchemeUsingGivenDevice_MostSpecificControl
             Is.EqualTo(ps4GamepadScheme));
         Assert.That(InputControlScheme.FindControlSchemeForDevice(xboxController, new[] { genericGamepadScheme, ps4GamepadScheme, xboxGamepadScheme, mouseScheme }),
             Is.EqualTo(xboxGamepadScheme));
+#endif
```

Comment:
The two tests here aren't really platform-specific. Would prefer an `[Ignore]` and case so I can fix this along with the other tests.

Reply:
Will do

Reply:
Done

Reply:
Awesome, thank you :)